src/actions/signup.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+
+import {
+    SIGNUP_SUCCESS,
+    SIGNUP_ERROR,
+    LOADING
+} from './action.types';
```

Comment:
Let's have the all the action types store in a separate folder called `types`. Just the same way we the actions and reducers